http/src/test/java/com/redhat/lightblue/client/http/LightblueHttpClientMock.java

```diff
@@ -1,25 +0,0 @@
-package com.redhat.lightblue.client.http;
-
-import org.apache.http.client.methods.HttpRequestBase;
-import com.redhat.lightblue.client.LightblueClientConfiguration;
-
-import com.redhat.lightblue.client.response.LightblueResponse;
-
-public class LightblueHttpClientMock extends LightblueHttpClient {
```

Comment:
Test subclass no longer needed because you can inject a `mock(HttpClient.class)`